test/ssl-tests/20-cert-select.cnf

```diff
@@ -1715,12 +1715,10 @@ client = 52-TLS 1.3 ECDSA with brainpool-client
 [52-TLS 1.3 ECDSA with brainpool-server]
 Certificate = ${ENV::TEST_CERTS_DIR}/server-ecdsa-brainpoolP256r1-cert.pem
 CipherString = DEFAULT
-Groups = brainpoolP256r1
```

Comment:
Is this still brainpool with the groups removed?

Reply:
Yes. This test is about cert selection which doesn't use the groups logic.